src/test/java/de/retest/recheck/review/workers/SaveFilterWorkerTest.java

```diff
@@ -0,0 +1,97 @@
+package de.retest.recheck.review.workers;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import org.junit.Rule;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.rules.TemporaryFolder;
+
+import de.retest.recheck.ignore.RecheckIgnoreLocator;
+import de.retest.recheck.review.GlobalIgnoreApplier;
+import de.retest.recheck.review.GlobalIgnoreApplier.PersistableGlobalIgnoreApplier;
+import de.retest.recheck.review.ignore.FilterPreserveLineLoader.FilterPreserveLine;
+
+class SaveFilterWorkerTest {
```

Comment:
You do IO, isn't that technically an integration test?

Reply:
Well ... since the unit is an IO unit ... how would you do it instead?

Reply:
I would just rename this to `SaveFilterWorkerIT` 😄

Reply:
Done